frontend/src/pages/pipelines/GlobalModelCustomizationRoutes.tsx

```diff
@@ -10,7 +10,7 @@ const GlobalModelCustomizationRoutes: React.FC = () => (
   <ProjectsRoutes>
     <Route path="/" element={<GlobalModelCustomization />} />
     <Route
-      path="instructlab/:namespace?/*"
+      path="fine-tune/:modelRegistry/:registeredModelId/:modelVersionId/:namespace"
```

Comment:
Should this not be in the routes file too?

Reply:
sure